conf/cassandra.yaml

```diff
@@ -1607,3 +1607,6 @@ enable_drop_compact_storage: false
 #         disallowed: []
 # Guardrail to allow/disallow user-provided timestamps. Defaults to true.
 #     user_timestamps_enabled: true
+# Guardrail to allow/disallow list operations that require read before write, i.e. setting list element by index and
+# removing list elements by either index or value. Defaults to true.
+#     read_before_write_list_operations_enabled: true
```

Comment:
Shouldn't it be disabled by default? Same as every new feature we add?

Reply:
My bad, they are disabled globally

Reply:
The guardrail is also disabled locally. Here `true` means that read before write operations are enabled as usual, so the guardrail isn’t having any effect.